docs/docs/Release_Notes/3.0.0/3.0.0.md

```diff
@@ -3,11 +3,20 @@
 To install or upgrade the SDK, refer to instructions [here](/Getting_Started.md#installation).
 
 ## New & Improved
-* Added a `scratch_path` property on the [RemoteHost](/References/Classes/#remotehost) object for storage and debugging purposes.
+* Added the ability to define snapshot parameters in a [Snapshot Parameters Definition](/References/Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-schema).
+	* Provide end-users with configurable options prior to taking a snapshot.
+	* The options selected are provided as input to pre/post-snapshot functions.
+
+* Added a `scratch_path` property on the [RemoteHost](/References/Classes/#remotehost) object which can be used to:
+	* Store small amounts of persistent data.
+	* Mount VDB data.
```

Comment:
"Mount VDB data" - not sure what this is saying. May be "VDB Mount Data" or "Mount data related to VDB"?

Reply:
Maybe...
> ... which can be used as:
>  * a place to store small amounts of persistent data
>  * a place to mount VDB data

Reply:
I'll take Tom's suggestion.